nixos/tests/pds.nix

```
@@ -0,0 +1,32 @@
+import ./make-test-python.nix (
+  { lib, ... }:
+  {
+    name = "PDS";
+
+    nodes.machine = {
+      services.pds = {
+        enable = true;
+        settings = {
+          PDS_PORT = "3000";
+          PDS_HOSTNAME = "example.com";
+
+          # DO NOT USE IN PRODUCTION
+          # Generate your secrets with
+          # `nix-build -A pds.passthru.generateSecrets && ./result/bin/generate-pds-secrets`
+          # and use `environmentFiles` to not store secrets in your /nix/store
```

This is the complete wrong place to document this. This must all go into the module.

```suggestion
          # Snake oil testing credentials
```